users.json

```diff
@@ -1,16 +1,16 @@
 [
   {
-    "username": "cramakri",
-    "firstName": "Chandrasekhar",
-    "lastName": "Ramakrishnan",
-    "email": "cramakri@ethz.ch",
-    "password": "cramakri"
+    "username": "sdrenku65535",
+    "firstName": "Foo",
+    "lastName": "Bar",
+    "email": "noreply@epfl.ch",
+    "password": "sdrenku-test1"
   },
   {
-    "username": "ableuler",
-    "firstName": "Andreas",
-    "lastName": "Bleuler",
-    "email": "andreas.bleuler@sdsc.ethz.ch",
-    "password": "ableuler"
+    "username": "sdrenku65534",
+    "firstName": "Foo2",
+    "lastName": "Bar2",
+    "email": "noreply@epfl.ch",
```

Comment:
Are you sure that keycloak does not enforce unique email addresses per user? I would assume so...

Reply:
@ableuler : not sure at all, closing this case anyway...